pkg/agent/openflow/pipeline_windows.go

```diff
@@ -23,6 +23,27 @@ import (
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 )
 
+// hostBridgeLocalFlows generates the flows to match the packets forwarded between bridge local port and uplink port.
+// IPAM is not currently supported on Windows, otherwise MatchDstMAC would need to be modified.
+func (f *featurePodConnectivity) hostBridgeLocalFlows() []binding.Flow {
+	cookieID := f.cookieAllocator.Request(f.category).Raw()
+	return []binding.Flow{
+		// This generates the flow to forward the packets from uplink port to bridge local port.
+		ClassifierTable.ofTable.BuildFlow(priorityNormal).
+			Cookie(cookieID).
+			MatchInPort(f.uplinkPort).
+			MatchDstIP(f.nodeConfig.NodeIPv4Addr.IP).
```

Comment:
I would prefer to use `f.nodeConfig.NodeTransportIPv4Addr.IP`because Antrea supports the scenario that the transport interface is different from Node_IP in the multiple NIC cases. See the usage in function `prepareHNSNetworkAndOVSExtension`.